charmcraft/commands/store/registry.py

```diff
@@ -38,6 +43,9 @@
     MANIFEST_V2_MIMETYPE,
 }
 
+# downloads and uploads happen in chunks
+CHUNK_SIZE = 2 ** 20
```

Comment:
why this size?

Reply:
This is mostly driven by the usage in the "upload blob" part, where I found that the cost in time was actually doing the dance to push bytes, not because of the quantity of bytes sent.

IOW (numbers are to exemplify), sending 64k takes 200ms, sending 1M takes 250ms.

Reply:
Sorry for being vague, I was hoping the reason for the size made it into the code comment :-)

Reply:
Added.